planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -67,11 +81,8 @@ PullOverModule::PullOverModule(
 
 void PullOverModule::resetStatus()
 {
-  status_.has_decided_path = false;
-  status_.path_type = PathType::NONE;
-  status_.is_safe = false;
-  status_.has_decided_velocity = false;
-  status_.has_requested_approval_ = false;
+  PUllOverStatus initial_status{};
```

Comment:
typo PUll -> Pull

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/1911/commits/b47d458c9870cfcb74d01d9866159bd273ee1972